CITATION.cff

```diff
@@ -0,0 +1,63 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Anzt"
+  given-names: "Hartwig"
+- family-names: "Chen"
+  given-names: "Yenchen"
+- family-names: "Cojean"
+  given-names: "Terry"
+- family-names: "Flegar"
+  given-names: "Goran"
+- family-names: "Göbel"
+  given-names: "Fritz"
+- family-names: "Grützmacher"
+  given-names: "Thomas"
+- family-names: "Kashi"
+  given-names: "Aditya"
+- family-names: "Koch"
+  given-names: "Marcel"
+- family-names: "Nayak"
+  given-names: "Pratik"
+- family-names: "Olenik"
+  given-names: "Gregor"
+- family-names: "Ribizel"
+  given-names: "Tobias"
+- family-names: "Riemer"
+  given-names: "Lukas"
+- family-names: "Tsai"
+  given-names: "Yuhsiang"
+title: "Ginkgo: A Modern Linear Operator Algebra Framework for High Performance Computing"
+version: 1.4.0
+date-released:
```

Comment:
should it be the release date?

Reply:
Yes, I wanted to do that, but missed it. Thanks